### ISSUE TITLE
Handling very long imports/processes

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/researcher-metadata#590]
Right now, the full Open Access Button import seems to take more than a week.  I'm not sure yet exactly how long it takes because I haven't gotten it to run to completion.  The full duplicate grouping task also takes over a week.  We'll need to find a way to better handle these.  We may just have to chunk them and run a quarter per week over the course of a month.  The OAB import could potentially be split up to have the imports with known DOIs (faster but more records) run every other week, and do the title query imports (slower but less records) on the in-between week, every other week.

The full grouping task doesn't have to be run very often, but it would be good to run it whenever we make changes to the grouping logic.

**Comment by ajkiessl** [comment]
The full OAB import took almost two weeks exactly.  I split it up in #612.

**Comment by ajkiessl** [comment]
Unpaywall solution for interruptions during the import: https://github.com/psu-libraries/researcher-metadata/pull/819

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Does this issue mean that we could be sending reminder emails about articles for which we haven't yet checked OA Button and/or Unpaywall?

**Comment by ajkiessl** [comment]
@anaelizabethenriquez No, we actually have a task that runs the OA Button and Unpaywall imports for all newly imported publications a few hours before emails go out.  We also aren't running any other imports from the time those "new" OA Button and Unpaywall imports happen, and when the emails go out.

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Thanks! I should have remembered that. I have this in the Backlog in the Fall-Winter 2024-2025 milestone. Please move it to the Backlog milestone if you think it should be lower priority.
